test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -723,6 +733,85 @@ UCS_TEST_P(test_ucp_sockaddr, err_handle) {
     EXPECT_EQ(1u, sender().get_err_num());
 }
 
+UCS_TEST_SKIP_COND_P(test_ucp_sockaddr, compare_cm_and_wireup_configs,
+                     no_close_protocol()) {
+    ucp_worker_cfg_index_t cm_ep_cfg_index, wireup_ep_cfg_index;
+    ucp_ep_config_key_t *cm_ep_cfg_key, *wireup_ep_cfg_key;
+
+    /* get configuration index for EP created through CM */
+    listen_and_communicate(false, SEND_DIRECTION_C2S);
+    /* TODO: remove the SKIP below and include for <uct/tcp/tcp_sockcm_ep.h>
+     *       header file, when CONNECT_TO_EP support is added for TCP */
+    if (sender().ep()->uct_eps[ucp_ep_get_cm_lane(sender().ep())]
+        ->iface->ops.ep_disconnect == uct_tcp_sockcm_ep_disconnect) {
+        UCS_TEST_SKIP_R("don't test TCP SOCKCM");
+    }
```

Comment:
can we check TCP by string name to avoid include?

Reply:
unfortunately, no
since the connection was already established and we don't know the name (it could be taken from cm_idx saved in cm_wireup_ep)

Reply:
can we use some uct-level query to check the name? or by cm_idx?

Reply:
done in another way: saved cm_idx to UCP EP config key